README.md

```diff
@@ -44,6 +44,7 @@ Options:
                      the current directory for one.
   -v, --verbosity    Set the verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and
                      diag[nostic]
+  --check            Terminate with non-zero exit code if any files formatted.

```

Comment:
❔ Why not make this "the number of files formatted"?

Reply:
In current implementation if `--check` option specified the application exits with `1` exit code, I did this like specified in `Prettier` documentation <https://prettier.io/docs/en/cli.html#check>. 

Do like to change the implementation to make `exit code=<number of files formetted>` when `--check` option is specified?

Reply:
> Do like to change the implementation to make exit code=number of files formatted...?

I would not be a primary consumer of this feature, so other people with more experience using a `--check` option would need to answer your question. Your response will point those users to related documentation though which is a nice help. 😄

Reply:
So lets leave it as it is or change is still needed?:)

Reply:
@roman-petrov 	I think the design that we doscussed for this is fine.